EvmYul/State/Account.lean

```diff
@@ -15,17 +15,19 @@ Suppose `a` is some address.
 In the yellow paper it is supposed to be a 256-bit hash of the root node of
 a Merkle Tree. KEVM implemets it as just an key/value map.
 - `storage`  -- σ[a]_s.
+- `tstorage`  -- Added in EIP-1153
 - `codeHash` -- σ[a]_c.
 
-For now, we assume no global map `GM` with which `GM[code_hash] ≡ code`. 
-- `code` 
+For now, we assume no global map `GM` with which `GM[code_hash] ≡ code`.
+- `code`
 -/
 structure Account :=
-  nonce    : UInt256 
+  nonce    : UInt256
   balance  : UInt256
   code     : ByteArray
   codeHash : UInt256 -- TODO - Probably not needed.
   storage  : Storage
+  tstorage  : Finmap (λ _ : UInt256 ↦ UInt256)
```

Comment:
So the normal storage is Address -> UInt256 and transactional storage is UInt256 -> UInt256? (The answer can be 'yuup' :grin:.)

Reply:
Aren't they both UInt256 -> UInt256?
`abbrev Storage : Type := Finmap (λ _ : UInt256 ↦ UInt256)`

Reply:
Oh, we have two, an additional one in `Conform` 🤦. We'll need to cleanup eventually :).